main/xfsprogs/APKBUILD

```diff
@@ -1,12 +1,12 @@
 # Contributor: Valery Kartel <valery.kartel@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=xfsprogs
-pkgver=4.19.0
-pkgrel=1
+pkgver=4.20.0
+pkgrel=0
 pkgdesc="XFS filesystem utilities"
 url="http://xfs.org/index.php/Main_Page"
 arch="all"
-license="LGPL"
+license="LGPL-2.1-or-later"
```

Comment:
Next time please mention if you do more than just bump, this so we know what to look for.

Thanks!

Reply:
Should be in title or can be in commit body ?